clients/src/main/java/org/apache/kafka/clients/producer/MockProducer.java

```diff
@@ -95,6 +97,27 @@ public MockProducer(boolean autoComplete, Partitioner partitioner, Serializer<K>
         this(Cluster.empty(), autoComplete, partitioner, keySerializer, valueSerializer);
     }
 
+    public void initTransactions() {
```

Comment:
Probably fine not to do implement these here, but perhaps add a JIRA to make sure we get to it eventually?